### PR TITLE
Start Daedalus on Windows without nix environment

### PR DESCRIPTION
**Pull request by danielmain**
This PR makes possible to run Daedalus on Windows without nix for testing purposes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,7 +151,28 @@ http://localhost:65432/metadata/query
...
 
-### Updating upstream dependencies (cardano-wallet, cardano-node, and iohk-nix)
+### Windows
+
+This option is only for troubleshooting windows specific issues with hardware wallets. It is not recommended to use Windows as a developer environment.
+Most of the commands need `nix` and will run only on Linux or macOS.
+
+#### Requisites
+- Windows 10/11
+- Daedalus testnet installation (similar version used in branch) in `C:\Program Files\Daedalus Testnet`
```
Comment (michalrus) [inline README.md:161]: I was wondering how you solved that. Nifty! :bulb:

**Review by michalrus — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (danielmain) [reply README.md:161]: 😅 We need an existing installation with the pre compiled cardano-wallet, cardano-node with all dlls needed.

Comment (michalrus) [reply README.md:161]: Clear :smile_cat:

File: README.md [hunk 2]
```diff
@@ -151,7 +151,28 @@ http://localhost:65432/metadata/query
...
+- NodeJS 16
+- Python2 *
+- `yarn global add windows-build-tools` *
```
Comment (renanvalentin) [inline README.md:164]: I think you can omit this step by extracting `daedalus\nix\windows-usb-libs.zip`  under `daedalus\build` folder.

**Review by renanvalentin — CHANGES_REQUESTED** [review]
awesome job @danielmain ! This will be really helpful for HW.  I left only a small comment regarding `daedalus\nix\windows-usb-libs.zip` that could solve the issue related to `node-gyp` . We could list it as an alternative solution

**Comment by danielmain** [comment]
This PR will continue on this https://github.com/input-output-hk/daedalus/pull/2942

**Pull request closed without merging**